awesome-books/modules/Date.js

```diff
@@ -0,0 +1,10 @@
+import { DateTime } from './luxon.js';
+
+// Date & Time Display Function
+const dateContainer = document.getElementById('date');
+const currentDate = DateTime.now();
+export default function displayDateTime() {
+  const dateString = currentDate.toLocaleString(DateTime.DATETIME_MED);
+  dateContainer.textContent = dateString;
+  setInterval(displayDateTime, 1000);
+}
```

Comment:
- [ ] Great job using `luxon` to display the time 💯 I just suggest to use arrow functions instead of function declaration
> For Example 💡
```js
const displayDateTime = () => {
  const dateString = currentDate.toLocaleString(DateTime.DATETIME_MED);
  dateContainer.textContent = dateString;
  setInterval(displayDateTime, 1000);
}

export default displayDateTime;
```